frontend/src/metabase/admin/datamodel/components/DimensionList/DimensionList.jsx

```diff
@@ -173,7 +176,14 @@ export class DimensionList extends Component {
 
     return (
       <FieldListGroupingTrigger
-        className="FieldList-grouping-trigger text-white-hover flex align-center p1 cursor-pointer"
+        className={cx(
+          QueryBuilderS.FieldListGroupingTrigger,
+          CS.flex,
+          CS.alignCenter,
+          CS.p1,
+          CS.cursorPointer,
+          "text-white-hover",
```

Comment:
nit: could you move `text-white-hover` to be between `FieldListGroupingTrigger` and `CS.flex` to preserve the class order?